src/commonMain/kotlin/exchange.dydx.abacus/processor/wallet/account/AccountProcessor.kt

```diff
@@ -299,13 +299,14 @@ internal open class SubaccountProcessor(parser: ParserProtocol) : BaseProcessor(
         modified = transform(modified, payload, "current", currentAccountKeyMap)
 
         if (firstTime) {
+            val subaccountNumber = parser.asInt(modified["subaccountNumber"])
```

Comment:
`parser.asInt` returns a nullable value, should we default to 0 here? if so, may want to make a helper function for this since we do this a lot in this PR.